lib/scan_result.dart

```diff
@@ -15,14 +15,26 @@ abstract class _ScanResultMetadata {
   static const String overflowServiceUuids = "overflowServiceUUIDs";
 }
 
+/// Type of items returned by [BleManager] during scanning.
```

Comment:
```suggestion
/// A scan result emitted by the scanning operation, containing [Peripheral] and [AdvertisementData]
```

Reply:
I added `.` to the end of this sentence. :)